bot.py

```diff
@@ -107,6 +107,11 @@ def get_members_by_role(role):
     if message.author == client.user:
         return
     member = server.get_member_named(str(message.author))
+    
+    #Remind Bryan he's a cuck
+    if message.author.id == "285262293053472768":
+        if random.randint(1, 10) == 1:
```

Comment:
I think 10% is possibly too high. Let's deploy with this, but we may need to tweak it downward later on.